packages/cli/src/ActiveWebhooks.ts

```diff
@@ -32,8 +32,11 @@ export class ActiveWebhooks {
 		activation: WorkflowActivateMode,
 	): Promise<void> {
 		if (workflow.id === undefined) {
```

Comment:
is this even possible now? I assume this is an old check, and since a month or so this check would always return false.

Reply:
Removed! I'd've expected TS to pick up on this...